docs/kit/logistics/kit-testing.md

```diff
@@ -0,0 +1,221 @@
+# Kit Testing
+
+Prior to SR2025, Kit Collation and Kit Testing were separate events.
+From SR2025, we have combined the two events to reduce the number of times that volunteers need to travel for kit events over the summer.
+The key improvements that have allowed for this are the increased automation of the test scripts and the included functionality of recording board asset codes during testing.
+This allows for the removal of the manual collation step.
+
+All the automated test scripts are included in the [kit-test-scripts package](https://github.com/srobo/kit-test-scripts) the wheel file can be downloaded from [the latest release](https://github.com/srobo/.
+kit-test-scripts/releases/latest/).
+
+
+Item | Testing Step
+--- | ---
+Brain Board | [Brain Board test below](#brain-board)
+Power Board | [Power Board test below](#power-board)
+Motor Board | [Motor Board test below](#motor-board)
+Servo Board | [Servo Board test below](#servo-board)
+Arduino | [Arduino test below](#arduino)
+USB Webcam | [Camera test below](#camera)
+Battery (LiPo) | Storage charged and manually inventoried
+Battery Charger | Storage charged and manually inventoried
+Charger Supply | Storage charged and manually inventoried
+Battery bag | Manually inventoried
+USB Memory Stick | Count & wipe
+USB A to B cable | Count and cable test
+Micro USB cable | Count and cable test
+USB Hub | Count only
+CamCon (7.5mm) | Count only
+MiniCamCon (5mm) | Count only
+MicroCamCon (3.81mm) | Count only
+Screwdriver | Count only
+Screw shields | Count only
+
+## Board Tests
+
+### Brain Board
+
+Here we are testing that the KCH power circuit and that the Pi can boot and execute robot code.
+
+For this test you will need:
+
+- A 12 volt power supply with adapter to Camcon.
+- A USB stick
+
+```python
+from pathlib import Path
+import RPi.GPIO as GPIO
+asset_file = Path("/proc/device-tree/hat/custom_0")
+if asset_file.exists():
+    asset = asset_file.read_text()
+    # append code to seen file
+    with open("tested_KCHs.txt", "a") as f:
+        f.write(asset)
+        f.write("\n")
+
+    GPIO.setmode(GPIO.BCM)
+    GPIO.setup([10, 23, 18], GPIO.OUT, initial=GPIO.HIGH)
```

Comment:
Question: What is this doing? If it's to indicate the output, I think letting the code error and using the "Ok" LED would be fine.

Reply:
We want to check the LEDs are still functional.